gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/client.py.j2

```diff
@@ -333,7 +333,9 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
         metadata = tuple(metadata) + (
             gapic_v1.routing_header.to_grpc_metadata((
                 {%- for field_header in method.field_headers %}
+                {%- if not method.client_streaming %}
```

Comment:
I was under the impression that headers for streaming methods don't make a lot of sense because they're not tightly bound to http semantics. If that's not correct, then I think we would need to ask someone who knows better.